src/plot_api/helpers.js

```diff
@@ -299,7 +299,7 @@ exports.cleanData = function(data, existingData) {
             trace.scene = Plots.subplotsRegistry.gl3d.cleanId(trace.scene);
         }
 
-        if(!Registry.traceIs(trace, 'pie')) {
+        if(!Registry.traceIs(trace, 'pie') && !Registry.traceIs(trace, 'bar')) {
```

Comment:
no need to clean (plotly.js lingo for backward-compatible mappings) attribute that have never been released.